index.html

```diff
@@ -24,257 +24,305 @@
-          </div>
-
-          <div class="card__details">
-
-            <h3 class="card__text card__text--about">Skills</h3>
+        <div class="about__btn">
+          <a href="#" class="btn btn--green">get me resume</a>
+        </div>
 
-            <ul class="about__card__list">
-              <li>Github</li>
-              <li>Git</li>
-              <li>Terminal</li>
-              <li>Figma</li>
-              <li>Design</li>
-            </ul>
+        <ul class="card-container">
 
-          </div>
-        </li>
-      </ul>
-    </section>
+          <li class="about__card">
 
+            <div class="about__card__imgbox">
+              <img src="images/icon-languages.svg" alt="Languages-icon" class="about__card__img">
+            </div>
+
+            <div class="card__details">
+
+              <h3 class="card__text card__text--about">Languages</h3>
+
+              <ul class="about__card__list">
+                <li>javaScript</li>
+                <li>Ruby</li>
+                <li>Python</li>
+                <li>html</li>
+              </ul>
+
+            </div>
+          </li>
+
+          <li class="about__card">
+
+            <div class="about__card__imgbox">
+              <img src="images/icon-frameworks.svg" alt="frameworks-icon" class="about__card__img">
+            </div>
+
+            <div class="card__details">
+
+              <h3 class="card__text card__text--about">Frameworks</h3>
+
+              <ul class="about__card__list">
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li>Angular</li>
+              </ul>
+
+            </div>
+          </li>
+
+          <li class="about__card">
+
+            <div class="about__card__imgbox">
+              <img src="images/icon-skills.svg" alt="skills-icon" class="about__card__img">
+            </div>
+
+            <div class="card__details">
+
+              <h3 class="card__text card__text--about">Skills</h3>
+
+              <ul class="about__card__list">
+                <li>Github</li>
+                <li>Git</li>
+                <li>Terminal</li>
+                <li>Figma</li>
+                <li>Design</li>
+              </ul>
+
+            </div>
+          </li>
+        </ul>
+      </section>
+    </main>
+    <footer class="contact">
+
+      <h3 class="title">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h3>
+
+      <form action="https://formspree.io/f/xzbopkkk" method="POST" id="my-form">
+
+        <input type="text" placeholder="Full name" maxlength="30" name="full-name" required>
+
+        <input type="email" placeholder="Email address" name="email" required>
+
+        <textarea placeholder="Enter text here" name="message" maxlength="500" required>  </textarea>
+
+        <input type="submit" class="btn btn--green" value="Get in Touch">
+      </form>
+
```

Comment:
- [ ] One of the major requirements is that your buttons have `hover`, `active`, and `focus` interactions with the user of your site. Kindly implement it by choosing a recommended color for these interactions. The colors to use can be found in this section of the Figma template:

![image](https://user-images.githubusercontent.com/77106962/144300156-98b496a8-c07b-47db-9fee-2e409896a330.png)

Reply:
I've already done that too and it works fine for me.

I implemented it in the 'btn' class, you can find it in the "styles/main.css" file at line 93, and applied hover and active

- This is my code 

         .btn:hover {
                   box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
                  }
         .btn:active {
                background-color: #008552;
              }

Reply:
I saw the one with the `box-shadow`, but I think it would be nice to use obvious ones. Remember your UI design should be interactive and obvious when users visit your site.